apps/challenges/challenge_config_utils.py

```diff
@@ -413,8 +459,13 @@ def validate_challenge_config_util(
         return error_messages, yaml_file_data, files
 
     phase_ids = []
+    phase_codenames = []
     files["challenge_test_annotation_files"] = []
     for data in challenge_phases_data:
+        if data["codename"] not in phase_codenames:
+            phase_codenames.append(data["codename"])
+        else:
+            error_messages.append("Found codename `{}` in phase `{}` which in not unique. Please use unique codenames for all phases.".format(data["codename"], data["name"]))
```

Comment:
change to `ERROR: Duplicate codename {} for phase {}. Please ensure codenames are unique`